src/content/docs/tutorial-infrastructure-monitoring/investigate-spike-CPU.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Investigate a spike in CPU
+metaDescription: Solve system outages by investigating resource bottlenecks across your system infrastructure.
+---
+
+import selectApps from 'images/infrastructure_screenshot-full_hosts-app-tile.webp'
+
+import updatedSummary from 'images/infrastructure_screenshot-full_updated-summary.webp'
+
+import appsSummaryTable from 'images/infrastructure_screenshot-full_hosts-updated-summary-table.webp'
+
+You can use the infrastructure hosts pages to troubleshoot different incidents within your system. In the last doc, we discussed the benefits of fully instrumenting your system with recommendations from the guided install, then walked through a general example of how and why you might filter down to a set of hosts. This doc builds off that skill and walks you through how you might troubleshoot a real-world example.  
+
+## Objectives [#objectives]
+
+In this tutorial, you'll learn how to:
+
+* Find the cause of an unexpected load spike
+* Determine if resources are deprived because of a new deployment, or because of a job process
+
+## Investigate a sudden spike in load [#spike-load]
+
+Let's say your customer support notified you that a certain part of your site is down, namely the ability to complete purchase transactions. You may not have all the details, but two possibilities may come to mind:
+
+* A new service deployment could be adding load to the servers or databases
+* A redundant background job process could be causing the spike 
+
+<Steps>
+    <Step>
+    ### Start with a filter set
+        
+Your system might have an excess of a thousand hosts, so you'll want to first filter down to a more specific set. Let's assume you have no additional context from an alert beyond what customer support relayed to you. You know that part of the site is down that covers purchase transactions, so you might start with the **Applications** tile to update your filter. There are five potential apps that could cause the site to go down.
```

Comment:
>There are five potential apps that could cause the site to go down.

I would remove this part. In the customer environment there may only be 1, plus this requires tribal knowledge of your app stack. If you don't know which services contribute to the checkout page then... I think the point is taken that filtering is a good first step without putting the user into a potential rabbit hole trying to translate our demo environment into their UI.